content/_layouts/corebaseline.html.haml

```diff
@@ -3,17 +3,20 @@ layout: developersection
 ---
 
 :ruby
-    oldest_weekly = site._generated[:core_tiers].weeklyCores[0]
+    oldest_supported_weekly = site._generated[:core_tiers].weeklyCores[0]
```

Comment:
Is there a reason for the variable renames? Makes the diff harder to follow.

Reply:
Undone for now.